diff -Nru sdcc-4.4.0/support/cpp/gcc/Makefile.in sdcc-4.4.0-patched/support/cpp/gcc/Makefile.in
--- sdcc-4.4.0/support/cpp/gcc/Makefile.in	2024-03-14 10:43:24.991437700 +0100
+++ sdcc-4.4.0-patched/support/cpp/gcc/Makefile.in	2024-03-15 12:40:21.669073000 +0100
@@ -68,7 +68,7 @@
 accel_dir_suffix = @accel_dir_suffix@
 
 # Sed command to transform gcc to installed name.
-program_transform_name := s,^,sd, # @program_transform_name@
+program_transform_name := @program_transform_name@
 
 # -----------------------------
 # Directories used during build
@@ -829,7 +829,7 @@
 # Actual name to use when installing a native compiler.
 GCC_INSTALL_NAME := $(shell echo gcc|sed '$(program_transform_name)')
 GCC_TARGET_INSTALL_NAME := $(target_noncanonical)-$(shell echo gcc|sed '$(program_transform_name)')
-CPP_INSTALL_NAME := $(shell echo cpp|sed '$(program_transform_name)')
+CPP_INSTALL_NAME := $(shell echo sdcpp|sed '$(program_transform_name)')
 GCOV_INSTALL_NAME := $(shell echo gcov|sed '$(program_transform_name)')
 GCOV_TOOL_INSTALL_NAME := $(shell echo gcov-tool|sed '$(program_transform_name)')
 GCOV_DUMP_INSTALL_NAME := $(shell echo gcov-dump|sed '$(program_transform_name)')
